assets/index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;400;500;600&display=swap" rel="stylesheet">
+    <title>Interactive Rating Component</title>
+</head>
+<body>
+    <!-- Section to hold the cards -->
+    <section class="rate-page">
+        <!-- Rate Card Start -->
+        <div class="rate-card">
+            <div class="circle">
+                <div class="icon-star">
+                    <img src="./img/icon-star.svg" alt="">
+                </div>
+            </div>
+            
+            
+            <div class="card-heading">
+                <h1>How did we do?</h1>
+            </div>
+            <div class="card-text">
```

Comment:
```suggestion
            <p class="card-text">Please let us know how we did with your support request. All feedback is appreciated to help us improve in our offering!</p>
```

Same here. You do not need the extra div for the font styling - you can apply directly to the element